README.md

```diff
@@ -80,6 +80,23 @@ steps:
       cake-bootstrap: true
 ```
 
+### `script-arguments`
+
+If you need pass same additional [argument](https://cakebuild.net/docs/writing-builds/args-and-environment-vars) value to CakeBuild, you can use `script-arguments`. 
+
+When you add an Argument: eg. 
+`var assemblyVersion = Argument<string>("versionTag", "refs/tag/v0.0.0.0");`
+then your step can look like that:
+
+```yml
+steps:
+  - name: Update-Assembly-Version
+    uses: jwickowski/cake-action@v0.0.2
+    with:
+      target: "Update-Assembly-Version"
+      script-arguments: --versionTag=${{ github.ref }}
```

Comment:
Question, are arguments quoted? Are quotes preserved?

Should example be `--versionTag="${{ github.ref }}" `?

Reply:
I have added test cases for that. 
Can you check main.test.ts line 121, 

`describe('When running the action with multiple script-arguments', () => {
  const fakeGetInput = core.getInput as jest.MockedFunction<typeof core.getInput>;
  const fakeRunScript = cake.runScript as jest.MockedFunction<typeof cake.runScript>;

  test('it should run script with the specified arguments', async () => {
    when(fakeGetInput)
      .calledWith('script-arguments').mockReturnValue('--firstInput=1.0.1 --secondInputQuotes="test value" --secondArg');

    await run();
    expect(fakeRunScript.mock.calls[0][4]).toStrictEqual(
      new CakeArgument('firstInput', '1.0.1'));

    expect(fakeRunScript.mock.calls[0][5]).toStrictEqual(
      new CakeArgument('secondInputQuotes', 'test value'));

    expect(fakeRunScript.mock.calls[0][6]).toStrictEqual(
      new CakeSwitch('secondArg'));
  });
});`